app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -31,39 +31,35 @@ def send_ready_to_verify_email(user, pii, enrollment)
         end
       end
 
-      def establishing_in_person_enrollment_for_user(user)
-        enrollment = user.establishing_in_person_enrollment
-        return enrollment if enrollment.present?
-
-        InPersonEnrollment.create!(user: user, profile: nil)
-      end
-
-      def usps_proofer
-        if IdentityConfig.store.usps_mock_fallback
-          UspsInPersonProofing::Mock::Proofer.new
-        else
-          UspsInPersonProofing::Proofer.new
-        end
-      end
-
       def create_usps_enrollment(enrollment, pii)
+        address = [pii['address1'], pii['address2']].select(&:present?).join(' ')
         applicant = UspsInPersonProofing::Applicant.new(
           {
             unique_id: enrollment.usps_unique_id,
             first_name: pii['first_name'],
             last_name: pii['last_name'],
-            address: pii['address1'],
-            # do we need address2?
+            address: address,
```

Comment:
Based on some of the testing feedback we've received, I'm hopeful this change may help fix some issues we've seen with apartment / unit numbers!

- https://gsa-tts.slack.com/archives/C03R8LXFPHR/p1659991062599009?thread_ts=1659990849.775479&cid=C03R8LXFPHR
- https://gsa-tts.slack.com/archives/C03R8LXFPHR/p1660062918778159?thread_ts=1660062196.659789&cid=C03R8LXFPHR